frontend/package.json

```diff
@@ -124,7 +126,7 @@
     "test": "react-scripts test --env=jsdom-fourteen --ci --watchAll=false --runInBand",
     "test:watch": "react-scripts test --env=jsdom-fourteen",
     "coverage": "react-scripts test --ci --coverage --testResultsProcessor jest-sonar-reporter --env=jsdom-fourteen --watchAll=false --runInBand",
-    "update-snapshots": "react-scripts test -- --updateSnapshot",
+    "update-snapshots": "react-scripts test --env=jsdom-fourteen --updateSnapshot",
```

Comment:
What's the `env` for?

Reply:
try running the old version, it won't work. the jsdom-fourteen has some required features we need to make the snapshots work.